### DISCUSSION TITLE
Welcome to community Discussions!

### DISCUSSION THREAD

**Discussion by amarts** [gluster/community#47] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by amarts** [comment]
I will start with myself. I am Amar Tumballi, currently maintainer of gluster projects. Also started a separate project, https://kadalu.io along with @aravindavk and few others to make gluster ready for new-gen technologies and management layers.

Here is my suggestion. Use this Discussions as a replacement for `gluster-users@` ML and Discussions in glusterfs repo as replacement to `gluster-devel@` ML. Let me know your thoughts.

**Comment by schaffung** [comment]
Hi Everyone!
I'm Srijan, I contribute to gluster project, mainly to glusterfs, glusto and glusterdocs repos.

+1 to @amarts suggestion. I feel we can effectively leverage this feature in github. Many a times queries are directed as issues in gh, which can actually be addressed here as a discussion.

**Comment by nik-redhat** [comment]
Hello Everyone!
I am Nikhil and I am mostly involved in contributing to [glusterfs](https://github.com/gluster/glusterfs), [glusto-tests](https://github.com/gluster/glusto-tests), and sometimes to [glusterdocs](https://github.com/gluster/glusterdocs) as well.

Please use this discussion room to get the answer for any of your queries related to gluster :slightly_smiling_face:

**Comment by tuaris** [comment]
Hello,

I'm Daniel and I have only started to get involved.  Earlier this year I 
contributed a few fixes for running GlusterFS on FreeBSD with help from 
Amar.  I am still learning about how things work with the hope of being 
able to contribute more in the upcoming year.
